frontend/public/src/containers/pages/login/LoginPages.js

```diff
@@ -26,7 +27,8 @@ const ForgotPasswordPages = () => (
 
 const LoginPages = () => (
   <Switch>
-    <Route exact path={routes.login.begin} component={LoginEmailPage} />
+    <Route exact path={routes.login.begin} component={LoginChoicePage} />
```

Comment:
I don't know that we want to present UI to the user to choose which way to login, but it should instead be feature flagged as still being `LoginEmailPage` unless the flag is enabled. If the flag is enabled then the "Login" button should send them directly to Keycloak to login.

Reply:
Refactored this so there's no choice page. I left the Create Account button, but also set its URL to the login URL - thinking that should probably be collapsed into Sign In as a Sign In / Create Account but also feel like that needs some design input.